cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -1,13 +1,17 @@
 {% extends "v1/layouts/layout-2-1.html" %}
 
 {% from 'tccp/macros/data_published.html' import data_published %}
+{% from 'tccp/macros/fields.html' import apr, apr_range %}
 
 {% block css%}
     {{ super() }}
     <style>
         dt {
             margin-top: 15px;
         }
+        dt:last-of-type {
+            margin-bottom: 0.5em;
+        }
         dt,
         dd {
             display: block;
```

Comment:
Is the plan to move these inline styles to a file some time?

Reply:
Yes, @contolini created a TCCP app in the frontend with a Less file [here](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/apps/tccp/css/main.less). These styles could be moved there either now or once the design of this page is finished in future.